src/test/test_verify_regression.cpp

```
@@ -0,0 +1,32 @@
+    auto raw_progs = verify_test::read_elf_cached("test-data/regression/map_sequential_lookup_unsafe.o", ".text", "",
+                                                  {}, &prevail::g_ebpf_platform_linux);
+    REQUIRE(raw_progs.size() == 1);
+    auto& raw_prog = raw_progs[0];
+    bool rejected = false;
+    try {
+        const auto prog_or_error = prevail::unmarshal(raw_prog, {});
+        const auto inst_seq = std::get_if<prevail::InstructionSeq>(&prog_or_error);
+        if (!inst_seq) {
+            rejected = true;
+        } else {
+            const prevail::Program prog = prevail::Program::from_sequence(*inst_seq, raw_prog.info, {});
```

Cppcheck (2.21.0)

[style] The function 'make' is never used.

(unusedFunction)